src/main/cliapp/src/components/Dashboard.js

```diff
@@ -76,21 +64,21 @@ export const Dashboard = () => {
 
             <div className="p-col-12 p-md-6 p-xl-3">
                 <div className="highlight-box">
-                    <div className="initials" style={{ backgroundColor: '#007be5', color: '#00448f' }}><span>ECO</span></div>
+                    <div className="initials" style={{ backgroundColor: '#007be5', color: '#00448f' }}><span>TV</span></div>
                     <div className="highlight-details ">
                         <i className="pi pi-search"></i>
-                        <span>Total Term Count</span>
-                        <span className="count">{ECOCount}</span>
+                        <span>Total Queries</span>
+                        <span className="count">523</span>
                     </div>
                 </div>
             </div>
             <div className="p-col-12 p-md-6 p-xl-3">
                 <div className="highlight-box">
-                    <div className="initials" style={{ backgroundColor: '#ef6262', color: '#a83d3b' }}><span>DO</span></div>
+                    <div className="initials" style={{ backgroundColor: '#ef6262', color: '#a83d3b' }}><span>TI</span></div>
                     <div className="highlight-details ">
                         <i className="pi pi-question-circle"></i>
-                        <span>Total Term Count</span>
-                        <span className="count">{DOCount}</span>
+                        <span>Total Issues</span>
+                        <span className="count">81</span>
```

Comment:
@adamgibs I am thinking that this change in this file was unintended? as it undoes my last commit?